include/mc_rbdyn/RobotConverter.h

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2022 CNRS-UM LIRMM, CNRS-AIST JRL
+ */
+
+#pragma once
+#include <mc_rbdyn/Robot.h>
+#include <mc_rbdyn/api.h>
+
+namespace mc_rbdyn
+{
+
+/**
+ * @brief Configuration for mc_rbdyn::RobotConverter
+ */
+struct RobotConverterConfig
+{
+  ///< Copy input robot's mbc to their corresponding output robot's mbc
+  bool mbcToOutMbc_ = true;
```

Comment:
Not necessarily directly related to the gripper PR, but we could probably find use-cases where this is useful. One example that comes to mind is the `EncoderObserver` that basically does that with the real robot. It can either copy joint values/velocities from the control mbc, or the encoder sensors. We could probably rewrite it using `RobotConverter` if we have sufficient granularity of options in `RobotConverter`.